javascript/code-challenges/challenges-05.test.js

```diff
@@ -0,0 +1,371 @@
+'use strict';
+
+const { last } = require("cheerio/lib/api/traversing");
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 1 - Review
+
+Write a function that iterates over an array of people objects
+and creates a new list of each person's full name using the array method 'map'.
+Each object will have the shape {firstName:string, lastName:string}
+E.g. [ { firstName:"Jane", lastName:"Doe" }, { firstName:"James", lastName:"Bond"}]
+should convert to ["Jane Doe", "James Bond"]
+Note the space in between first and last names.
+You can assume that neither firstName nor lastName will be blank
+------------------------------------------------------------------------------------------------ */
+const toLastNames = people => {
+  return people.map(fullName => {
+    return fullName.firstName + " " + fullName.lastName;
+  });
+};
+
```

Comment:
don't forget back tics and template literals!
```js
return `${person.firstName} ${person.lastName}`;
```
Concat works too, and template literals doesn't give you any edge on speed etc. For me it just creates more readable code.